web-admin/src/routes/[organization]/[project]/+layout.svelte

```diff
@@ -63,7 +64,9 @@
 
   $: onProjectPage = isProjectPage($page);
   $: onPublicURLPage = isPublicURLPage($page);
-  $: if ($page.url.searchParams.has("token") && isPublicReportPage($page)) {
+  $: onPublicReportOrAlertPage =
+    isPublicReportPage($page) || isPublicAlertPage($page);
+  $: if ($page.url.searchParams.has("token") && onPublicReportOrAlertPage) {
```

Comment:
Nit: `page.url.searchParams.has("token")` is covered by the `isPublicAlert()` check